contracts/ICommunityResolver.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {ISchemaResolver} from "@ethereum-attestation-service/eas-contracts/contracts/resolver/ISchemaResolver.sol";
+
+interface ICommunityResolver is ISchemaResolver {
+    isAdmin(bytes32 communityUID, address addr);

```

Comment:
Did this even compile? It has lot of things missing.

Reply:
It does, the interface is an extension of ISchemaResolver and does not need to implement it.
This is used only to call isAdmin from milestone status resolver